frontend/server/src/DAO/QualityNominations.php

```diff
@@ -123,6 +123,70 @@ public static function reviewerHasQualityTagNominatedProblem(
         ) > 0;
     }
 
+    /**
+    * Returns the contents and id of a nomination for a given problem and user.
+    * @return array{contents: array{quality_seal: bool, tag: string}, qualitynomination_id: int}
+    */
+    public static function getReviewedData(
+        \OmegaUp\DAO\VO\Identities $identity,
+        \OmegaUp\DAO\VO\Problems $problem
+    ): array {
+        $sql = "
+            SELECT
+                qn.qualitynomination_id,
+                qn.contents
+            FROM
+                QualityNominations qn
+            INNER JOIN
+                Identities i ON i.user_id = qn.user_id
+            WHERE
+                nomination = 'quality_tag' AND
+                i.identity_id = ? AND
+                qn.problem_id = ?";
+
+        /** @var array{contents: string, qualitynomination_id: int}|null */
+        $query = \OmegaUp\MySQLConnection::getInstance()->GetRow(
+            $sql,
+            [$identity->identity_id, $problem->problem_id]
+        );
+
+        if (is_null($query)) {
+            return [];
```

Comment:
Acá va a faltar cambiar esto para que regrese `null`:
```suggestion
            return null;
```
esto es más legible que un interpretar un array vacío.